### PR TITLE
TAJO-1740: Update Partition Table document

### PR DESCRIPTION
**Pull request by blrunner**
Currently, Tajo doesn't provide enough informations about partition table. Thus, we need to add more informations to following documentation.

http://tajo.apache.org/docs/current/partitioning/column_partitioning.html

### CODE REVIEW

File: tajo-docs/src/main/sphinx/partitioning/add_data_to_partition_table.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  INSERT OVERWRITE INTO student
+  SELECT id, name, grade, country, city
+  FROM   student_source;
+
+
+================================================
+How to CTAS dynamically to partition table
+================================================
+
+If you want to load an entire country or an entire city in one fell swoop:
```
Comment (eminency) [inline tajo-docs/src/main/sphinx/partitioning/add_data_to_partition_table.rst]: Explanation is same precisely as above one.
I think it's better to add some words like 'including CREATE TABLE' or 'when a partition table is created'

File: tajo-docs/src/main/sphinx/partitioning/alter_partition.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+You can use ``ALTER TABLE DROP PARTITION`` to drop a partition for a table. This doesn't remove the data for partition table. But if ``PURGE`` is specified, the partition data will be removed. The metadata is completely lost in all cases. An error is thrown if the partition for the table doesn't exists. You can use ``IF EXISTS`` to skip the error.
+
+*Examples*
+
+.. code-block:: sql
+
+  -- Delete just metadata
+  ALTER TABLE table1 DROP PARTITION (country = 'KOREA' , city = 'SEOUL');
+  -- Delete table data and metadata
+  ALTER TABLE table1 DROP PARTITION (country = 'USA', city = 'NEWYORK' ) PURGE
```
Comment (eminency) [inline tajo-docs/src/main/sphinx/partitioning/alter_partition.rst]: ';' is missed

File: tajo-docs/src/main/sphinx/partitioning/define_partition_table.rst [hunk 3]
```diff
@@ -0,0 +1,73 @@
+*********************************
+Define Partition Table
+*********************************
+
+Tajo makes it easy to specify an automatic partition scheme when the table is created.
+
+================================================
+How to Create Partitione Table
+================================================
```
Comment (eminency) [inline tajo-docs/src/main/sphinx/partitioning/define_partition_table.rst]: 'Partitione' -> 'Partition'

File: tajo-docs/src/main/sphinx/partitioning/define_partition_table.rst [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+.. code-block:: sql
+
+  CREATE TABLE student (
+    id     INT,
+    name   TEXT,
+    grade  TEXT
+  ) USING PARQUET
+  PARTITION BY COLUMN (country TEXT, city TEXT);
+
+The statement above creates the student table with id, name, grade. The table is also partitioned and data is stored in parquet files.
```
Comment (eminency) [inline tajo-docs/src/main/sphinx/partitioning/define_partition_table.rst]: Is 'parquet files' right?

File: tajo-docs/src/main/sphinx/partitioning/define_partition_table.rst [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+* ``=``
+* ``<>``
+* ``>``
+* ``<``
+* ``>=``
+* ``<=``
+* LIKE predicates with a leading wild-card character
+* IN list predicates
+
+Now above example table data is partitioned by country and city, so when the query is applied on table it can easily access the required row by the help partitions
```
Comment (eminency) [inline tajo-docs/src/main/sphinx/partitioning/define_partition_table.rst]: 'the help of partitions' looks better

File: tajo-docs/src/main/sphinx/partitioning/hive_compatibility.rst [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+  type:COLUMN
+  columns::default.student.country (TEXT), default.student.city (TEXT)
+
+
+
+================================================
+How to add data to partition table
+================================================
+
+In Tajo, you can add data dynamically to partition table of Hive with both ``INSERT INTO ... SELECT`` and ``Create Table As Select (CTAS)`` statments. Tajo will automatically filter the data to HiveMetastore, create directories and move filtered data to appropriate directory on the distributed file system
```
Comment (eminency) [inline tajo-docs/src/main/sphinx/partitioning/hive_compatibility.rst]: '.' is missed at the end of last statement.

**Comment by eminency** [comment]
Hi, @blrunner .

I reviewed your writing documents and left some inline comments.

Please check them out.

**Comment by blrunner** [comment]
Hi @eminency 

Thank you for your detailed review.
Honestly, I've changed existing contents too much. So, I'll make new PR to keep existing ones. When making new PR, I'll must reference your feedback.

**Comment by hyunsik** [comment]
I recommend you to keep a single page for column partition. Each content is too short. Later, it may be hard to see all contents at once after we add new partition type.

**Comment by hyunsik** [comment]
I saw new PR. Please ignore the last comment.
https://github.com/apache/tajo/pull/728

**Pull request closed without merging**